lib/Platform/Intl/PlatformIntlApple.mm

```diff
@@ -4,20 +4,79 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
```

Comment:
There are a few of these deleted newlines in the PR, please remove them to avoid adding noise to the PR